src/Symfony/Bundle/TwigBundle/DependencyInjection/TwigExtension.php

```diff
@@ -70,7 +70,7 @@ public function load(array $configs, ContainerBuilder $container): void
             $container->removeDefinition('twig.translation.extractor');
         }
 
-        if ($container::willBeAvailable('symfony/validator', Constraint::class, ['symfony/twig-bundle'])) {
+        if ($container::willBeAvailable('symfony/twig-bridge', TwigValidator::class, ['symfony/twig-bundle'])) {
```

Comment:
`willBeAvailable` is whether a package is installed or not. Here the `twig-bridge` package is always installed, but `TwigValidator` is only available in version 7.3.

Maybe add both conditions:
```suggestion
        if (class_exists(TwigValidator::class) && $container::willBeAvailable('symfony/validator', Constraint::class, ['symfony/twig-bundle'])) {
```

Reply:
But it's more direct to require `symfony/twig-bridge: ^7.3` in `symfony/twig-bundle`

Reply:
> But it's more direct to require `symfony/twig-bridge: ^7.3` in `symfony/twig-bundle`

good point, let's do that instead